include/reflection-cpp/reflection.hpp

```diff
@@ -160,17 +160,54 @@ inline constexpr auto CountMembers = [] {
 
 constexpr size_t MaxReflectionMemerCount = 5; // should go as high as 256, I guess?
 
+
+/**
+
+elisp functions to fill the ToTuple function
```

Comment:
ugh, elisp? :dagger: 

Alright, ultimatively (in the long run), at lesat for the `Lightweight` SQL project, I probably need something way beyond 100. But not now for sure. I was considering something like a top and bottom marker in that file, and then a script the inject this unpleasant code fragment fest of repetition in between the two markers (discarding what used to be within them before). Hmm... But that's up for a future idea, I think :)

Reply:
This only becomes relevant when we eventually start using Lightweight at the company. So we have some time :)